File: src/com/ecci/Hamers/Beer.java
package com.ecci.Hamers;

public class Beer {

    private String name;
    private String soort;
    private String pictureURL;
    private String percentage;
    private String brewer;
    private String country;

    public Beer(String name, String soort, String pictureURL, String percentage, String brewer, String country) {
        super();
        this.name = name;
        this.soort = soort;
        this.pictureURL = pictureURL;
        this.percentage = percentage;
        this.brewer = brewer;
        this.country = country;
    }

    public String getName() {
        return name;
    }

    public String getSoort() {
        return soort;
    }

    public String getPictureUrl() {
        return pictureURL;
    }

    public String getPercentage() { return percentage; }

    public String getBrewer() { return brewer; }

    public String getCountry() { return country; }
}

File: hamersapp/src/main/java/nl/ecci/hamers/models/Beer.kt
package nl.ecci.hamers.models

import com.google.gson.annotations.SerializedName
import nl.ecci.hamers.utils.Utils
import java.util.*

class Beer {
    val id: Int = Utils.notFound
    var name: String = Utils.unknown
    @SerializedName("soort")
    var kind: String = Utils.unknown
    @SerializedName("picture")
    val imageURL: String = Utils.unknown
    var percentage: String = Utils.unknown
    var brewer: String = Utils.unknown
    var country: String = Utils.unknown
    @SerializedName("cijfer")
    var rating: String = Utils.unknown
    val url: String = Utils.unknown
    @SerializedName("created_at")
    val createdAt: Date = Date()

    companion object {
        val BEER = "BEER"
    }
}